react/CipherIcon/styles.styl

```diff
@@ -0,0 +1,17 @@
+@require 'settings/breakpoints'
+@require 'settings/palette'
+@require 'tools/mixins'
+
+.c-card {
+    border rem(1) solid var(--silver)
+    border-radius rem(8)
+    padding rem(16)
+    text-decoration none
+    display block
+}
```

Comment:
Nit: I think you can remove the curly braces

Reply:
Actually I forgot to remove this file after `cp` from another component

Reply:
This styles are not used in this PR right ?